web/src/pages/news/in-depth/index.page.tsx

```diff
@@ -0,0 +1,126 @@
+import { NextSeo } from "next-seo";
+import React from "react";
+import { StoryblokStory } from "storyblok-generate-ts";
+
+import { ActionBanner } from "@nice-digital/nds-action-banner";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
+import { Button } from "@nice-digital/nds-button";
+import { PageHeader } from "@nice-digital/nds-page-header";
+
+import { ErrorPageContent } from "@/components/ErrorPageContent/ErrorPageContent";
+import { FeaturedStory } from "@/components/Storyblok/News/FeaturedStory/FeaturedStory";
+import { NewsList } from "@/components/Storyblok/News/NewsList/NewsList";
+import { NewsListNav } from "@/components/Storyblok/News/NewsListNav/NewsListNav";
+import { NewsListPagination } from "@/components/Storyblok/News/NewsListPagination/NewsListPagination";
+import { NewsStory } from "@/types/News";
+import { validateRouteParams } from "@/utils/storyblok";
+
+import type { GetServerSidePropsContext } from "next";
+
+export type InDepthArticleProps = {
+	featuredStory?: StoryblokStory<NewsStory> | null;
+	stories: StoryblokStory<NewsStory>[];
+	total: number;
+	currentPage: number;
+	perPage: number;
+	error?: string | undefined;
+};
+
+export const InDepthArticleIndexPage = ({
+	stories,
+	currentPage,
+	total,
+	perPage,
+	featuredStory,
+	error,
+}: InDepthArticleProps): React.ReactElement => {
+	if (error) {
+		return <ErrorPageContent title="Error" heading={error} />;
+	}
+	return (
+		<>
+			<NextSeo title="In-depth" openGraph={{ title: "In-depth" }}></NextSeo>
+			<PageHeader
+				heading="In-depth"
+				variant="fullWidthDark"
+				lead="Our in-depth articles explore how our work is making a real difference to people's lives."
```

Comment:
It feels like some of this content belongs in the CMS - not something to sort immediately, but I'll stick it on the tech debt spreadsheet. There are a few pages I've added with a lot of hard-coded text too, so we should probably make a list

Reply:
Sounds sensible - another option would be to just fling in the associated component (e.g. hero or page header) and leave it there

One for us to look at when I'm back maybe! Shouldn't hold up the PR